kitsune/sumo/static/sumo/js/codemirror.sumo-hint.js

```diff
@@ -0,0 +1,28 @@
+(function($) {
+  "use strict";
+
+  CodeMirror.registerHelper("hint", "sumo", function(editor, options) {
+    options = $.extend({}, options, {word: /[\w$-]+/, range: 500});
```

Comment:
This will override the options passed, since `$.extend` makes later argument take precedence (according to the docs). It seemed last time like you wanted these values to be the defaults, not forced values.